CHANGES.rst

```diff
@@ -117,8 +117,10 @@ Other changes
 
 * The search order for include files was changed. Previously it was
   ``include_directories``, ``Cython/Includes``, ``sys.path``. Now it is
-  ``include_directories``, ``sys.path``, ``Cython/Includes``. (Github issue
-  #2905)
+  ``include_directories``, ``sys.path``, ``Cython/Includes``. This was done to
+  allow third-party `__index__.pxd` files to override the ones in Cython.
```

Comment:
`__index__.pxd` ? You probably meant to use some generic name like `special_library.pxd`, right?

Reply:
changed to `*.pxd`